aws-datastore/src/androidTest/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapterClearTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.datastore.storage.sqlite;
+
+import android.content.Context;
+import androidx.test.core.app.ApplicationProvider;
+
+import com.amplifyframework.datastore.DataStoreException;
+import com.amplifyframework.datastore.StrictMode;
+import com.amplifyframework.datastore.storage.StorageItemChange;
+import com.amplifyframework.datastore.storage.SynchronousStorageAdapter;
+import com.amplifyframework.testmodels.commentsblog.AmplifyModelProvider;
+import com.amplifyframework.testmodels.commentsblog.BlogOwner;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.List;
+
+import io.reactivex.observers.TestObserver;
+
+import static com.amplifyframework.datastore.storage.sqlite.SQLiteStorageAdapter.DATABASE_NAME;
```

Comment:
For unit tests, I think it is a better test not to share constants between test code and production code.

Reply:
^ I do agree. I made this change in my own tests probably 1-2 years ago.

It's a tradeoff of maintaining two copies of a constant, versus keeping the constant as an implementation detail (`private` field access, etc.) 

If the source code version changes, a self-maintained version in test would be able to catch it. But, if you just use the constant, you could have like:

```
@VisibleForTesting static final String GOOD_VALUE = "oopsActuallyBadPsych!";
```

If test still had a hardcoded `"goodValue"` somewhere, it'd catch it.

Reply:
Fixing this.